### DISCUSSION TITLE
opera compatability

### DISCUSSION THREAD

**Discussion by spydragon** [sawyerpollard/Minecraft-Weather-New-Tab#21] [Ideas]
Hello, I was wondering if this extension could be expanded to opera/opera GX
I would understand if your not willing.

**Comment by GreatSymphonia** [comment]
Duplicate of #6 
@spydragon

**Comment by GreatSymphonia** [comment]
In brief, it is simply not possible to natively modify the home page on opera browser, you need to do it manually.

**Reply by spydragon** [reply]
Hmm, that’s interesting, well thank you anyway.

**Reply by GreatSymphonia** [reply]
No problem, actually, there is a workaround (on the reddit post I linked in the issue) but I didn't test it.

**Reply by GreatSymphonia** [reply]
Found the reddit post:
[https://www.reddit.com/r/operabrowser/comments/nm12vj/new_tab_extensions_not_working_opera_gx/](https://www.reddit.com/r/operabrowser/comments/nm12vj/new_tab_extensions_not_working_opera_gx/)

**Reply by spydragon** [reply]
wow, the extension works great! but I don't think its working with yours, it needs a URL to work

**Reply by GreatSymphonia** [reply]
Ahh, true, maybe hosting the extension as a webpage on localhost would do, but at this point, I would simply switch browsers lol

**Reply by spydragon** [reply]
Haha, ya. Thanks for your help
